views/home.hbs

```diff
@@ -1,13 +1,142 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Please don't delete these lines this helps in adapting the website in mobile view...